### PR TITLE
Optimize rid-bags' serialization

### PR DESCRIPTION
**Pull request by markodjurovic**
Optimize size of rid-bags serialization result, by using varints instead of fixed length data fields

### CODE REVIEW

**Comment by lvca** [comment]
@markodjurovic are we keeping the previous serialization to open old databases?

File: core/src/main/java/com/orientechnologies/orient/core/storage/ridbag/sbtree/OSBTreeRidBag.java [hunk 1]
```diff
@@ -619,21 +621,17 @@ public Object returnOriginalState(List<OMultiValueChangeEvent<OIdentifiable, OId
...
-    return result;
+  public int getSerializedSize() {    
+    return 0;    
   }
 
-  @Override
-  public int getSerializedSize(byte[] stream, int offset) {
-    return getSerializedSize();
-  }
+//  @Override
```
Comment (lvca) [inline core/src/main/java/com/orientechnologies/orient/core/storage/ridbag/sbtree/OSBTreeRidBag.java]: You can get rid of old methods, without commenting them. We have github for the history ;-)

Comment (markodjurovic) [reply core/src/main/java/com/orientechnologies/orient/core/storage/ridbag/sbtree/OSBTreeRidBag.java]: You are right :)

**Comment by markodjurovic** [comment]
Invalid, not backward compatibile

**Pull request closed without merging**